### PR TITLE
Convert Oracle to Java Timestamp, update namespace names for dashboard

### PR DESCRIPTION
**Pull request by ryankwagner**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/namespace/RocksDBExtractionNamespace.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 import javax.validation.constraints.NotNull;
 import java.util.Objects;
 
-@JsonTypeName("maharocksdb")
+@JsonTypeName("mahainmemorydb")
```
Comment (patelh) [inline druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/namespace/RocksDBExtractionNamespace.java]: This is not a backwards compatible change.  What is the rationale for this change?

File: druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/namespace/ExtractionNamespace.java [hunk 2]
```diff
@@ -10,7 +10,6 @@
 @JsonSubTypes(value = {
         @JsonSubTypes.Type(name = "mahajdbcleaderfollower", value = JDBCExtractionNamespaceWithLeaderAndFollower.class),
         @JsonSubTypes.Type(name = "mahajdbc", value = JDBCExtractionNamespace.class),
-        @JsonSubTypes.Type(name = "maharocksdb", value = RocksDBExtractionNamespace.class),
```
Comment (patelh) [inline druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/namespace/ExtractionNamespace.java]: Not backwards compatible.  This was added for a reason, see #418

Comment (ryankwagner) [reply druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/namespace/ExtractionNamespace.java]: This line and the one after it don't agree with each other.  The registration in the class above (JsonTypeName) declares one name, while here we are using two (one of which is undeclared) so whenever an ExtractionNamespace calls whichever of the two names isn't mentioned in RocksDBExtractionNamespace.java the json type will come up as undeclared.  I changed it back to mahainmemorydb to keep it consistent with the current usage.

**Comment by ryankwagner** [comment]
Generated stacktrace from current:
`Historical [main] Error when starting up.  Failing.
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_181]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_181]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_181]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_181]
	at io.druid.java.util.common.lifecycle.Lifecycle$AnnotationBasedHandler.start(Lifecycle.java:414) ~[java-util-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	at io.druid.java.util.common.lifecycle.Lifecycle.start(Lifecycle.java:311) ~[java-util-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	at io.druid.guice.LifecycleModule$2.start(LifecycleModule.java:156) ~[druid-api-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	at io.druid.cli.GuiceRunnable.initLifecycle(GuiceRunnable.java:101) [druid-services-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	at io.druid.cli.ServerRunnable.run(ServerRunnable.java:50) [druid-services-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	at io.druid.cli.Main.main(Main.java:108) [druid-services-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
Caused by: io.druid.java.util.common.ISE: Exception during reading lookups from [/home/y/var/cdw_lookups/lookupSnapshot.json]
	at io.druid.query.lookup.LookupSnapshotTaker.pullExistingSnapshot(LookupSnapshotTaker.java:79) ~[druid-processing-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	at io.druid.query.lookup.LookupReferencesManager.getLookupListFromSnapshot(LookupReferencesManager.java:455) ~[druid-server-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	at io.druid.query.lookup.LookupReferencesManager.getLookupsList(LookupReferencesManager.java:366) ~[druid-server-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	at io.druid.query.lookup.LookupReferencesManager.loadAllLookupsAndInitStateRef(LookupReferencesManager.java:345) ~[druid-server-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	at io.druid.query.lookup.LookupReferencesManager.start(LookupReferencesManager.java:150) ~[druid-server-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	... 10 more
Caused by: com.fasterxml.jackson.databind.JsonMappingException: Could not resolve type id 'mahainmemorydb' into a subtype of [simple type, class com.yahoo.maha.maha_druid_lookups.query.lookup.namespace.ExtractionNamespace]
 at [Source: <our snapshot dir>; line: 1, column: 3971] (through reference chain: java.util.ArrayList[3])
	at com.fasterxml.jackson.databind.JsonMappingException.from(JsonMappingException.java:148) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.DeserializationContext.unknownTypeException(DeserializationContext.java:862) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.jsontype.impl.TypeDeserializerBase._findDeserializer(TypeDeserializerBase.java:167) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.jsontype.impl.AsPropertyTypeDeserializer._deserializeTypedForId(AsPropertyTypeDeserializer.java:99) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.jsontype.impl.AsPropertyTypeDeserializer.deserializeTypedFromObject(AsPropertyTypeDeserializer.java:84) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.AbstractDeserializer.deserializeWithType(AbstractDeserializer.java:132) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:536) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:344) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1064) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:264) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeOther(BeanDeserializer.java:156) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:126) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.jsontype.impl.AsPropertyTypeDeserializer._deserializeTypedForId(AsPropertyTypeDeserializer.java:113) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.jsontype.impl.AsPropertyTypeDeserializer.deserializeTypedFromObject(AsPropertyTypeDeserializer.java:84) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.AbstractDeserializer.deserializeWithType(AbstractDeserializer.java:132) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:536) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:344) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1064) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:264) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:124) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:232) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:206) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:25) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:3066) ~[jackson-databind-2.4.6.jar:2.4.6]
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2122) ~[jackson-databind-2.4.6.jar:2.4.6]
	at io.druid.query.lookup.LookupSnapshotTaker.pullExistingSnapshot(LookupSnapshotTaker.java:75) ~[druid-processing-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	at io.druid.query.lookup.LookupReferencesManager.getLookupListFromSnapshot(LookupReferencesManager.java:455) ~[druid-server-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	at io.druid.query.lookup.LookupReferencesManager.getLookupsList(LookupReferencesManager.java:366) ~[druid-server-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	at io.druid.query.lookup.LookupReferencesManager.loadAllLookupsAndInitStateRef(LookupReferencesManager.java:345) ~[druid-server-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	at io.druid.query.lookup.LookupReferencesManager.start(LookupReferencesManager.java:150) ~[druid-server-0.11.1-1515709212-d5fa1b6-456.jar:0.11.1-1515709212-d5fa1b6-456]
	... 10 more
`

And the actual underlying issue came from the framework attempting to boot from a previous snapshot using the current version:
`
Coordinator is unavailable. Loading saved snapshot instead
`

That snapshot attempted to load a subclass of extractionNamespace (from 6 months ago) which was expecting InMemoryDBExtractionNamespace.class (which no longer exists).  The SerDe for maharocksdb is back, and I'm going to add a test to assert both versions are working.
